freqtrade/commands/list_commands.py

```diff
@@ -36,22 +38,41 @@ def start_list_exchanges(args: Dict[str, Any]) -> None:
             print(f"Exchanges available for Freqtrade: {', '.join(exchanges)}")
 
 
+def _print_objs_tabular(objs: List, print_colorized: bool) -> None:
+    if print_colorized:
+        colorama_init(autoreset=True)
+
+    names = [s['name'] for s in objs]
+    objss_to_print = [{
+        'name': s['name'] if s['name'] else "--",
+        'location': s['location'].name,
+        'status': (((Fore.RED if print_colorized else '') +
+                    "LOAD FAILED" + (Style.RESET_ALL if print_colorized else ''))
+                   if s['class'] is None
+                   else "OK" if names.count(s['name']) == 1
+                   else ((Fore.YELLOW if print_colorized else '') +
+                         "DUPLICATE NAME" +
+                         (Style.RESET_ALL if print_colorized else '')))
```

Comment:
personally, i'd prefer to extract the `if print_colorized` out of the list-comprehension as follows to reduce the complexity of the comprehension.

``` python
red = Fore.RED if print_colorized else ''
warn = Fore.YELLOW  if print_colorized else ''
col_reset = Style.RESET_ALL  if print_colorized else ''
```

We can then use the colors (if enabled) - or simply have an empty string ... which will disable it automatically.